articles/laravel-8-email-verification/index.md

```diff
@@ -46,24 +46,28 @@ cd verifyEmailApp
 php artisan serve
 ```
 
+`php artisan serve` command allows the application to be hosted locally.
+
 **Using Laravel Installer**  
-To build Laravel applications easily, Laravel provides a helpful command-line utility. Download the installer first: 
+There are numerous command-line utilities that make it easy to build applications using Laravel. We use the following command to download the Laravel installer: 
 
```

Comment:
In this part, the command-line utility in question is 'Laravel installer', changing this to 'numerous utilities  would raise questions!

Reply:
Is this part of the plagiarism? Also, Composer is a command line utility - numerous utilities is justified.  The second statement specifies the specific utility.